mavros_extras/CMakeLists.txt

```diff
@@ -109,7 +110,7 @@ install(PROGRAMS
 )
 
 ## Mark executables and/or libraries for installation
-install(TARGETS mavros_extras copter_visualization servo_state_publisher
+install(TARGETS mavros_extras visualization servo_state_publisher
```

Comment:
> anything else.

Not true. It models waypoints also and now it can model landing targets. The model is a quadcopter, but in the end this is a visualization node for several features. more than just a multicopter. It means we can generalize it also to add other models. So I agree with this change.